zulip/integrations/hg/zulip-changegroup.py

```diff
@@ -27,11 +27,14 @@
 # `hg push`). See https://zulipchat.com/integrations for installation instructions.
 from __future__ import absolute_import
 
+# disable mercurial demandimport or zulip breaks when importing
+from mercurial import demandimport
+demandimport.disable()
+
 import zulip
 import sys
 from six.moves import range
 from typing import Any, Optional, Text
-from mercurial import ui, repo
```

Comment:
@scottrus it is wrong to remove importing of these modules, because they are required in the following code execution